services/movie_session.py

```diff
@@ -42,3 +41,12 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Here you are using `get()` method which will raise a `DoesNotExist` exception if no object found. It would be better to use `filter()` method along with `delete()` which will not raise an exception if no object found.